File: .NET/Microsoft.Recognizers.Text.Options/Config/BooleanParserConfiguration.cs
using System.Collections.Generic;

namespace Microsoft.Recognizers.Text.Options
{
    public class BooleanParserConfiguration : IOptionParserConfiguration<bool>
    {
        public static IDictionary<string, bool> Resolutions = new Dictionary<string, bool>
        {
            { Constants.SYS_BOOLEAN_TRUE, true },
            { Constants.SYS_BOOLEAN_FALSE, false }
        };

        IDictionary<string, bool> IOptionParserConfiguration<bool>.Resolutions => Resolutions;
    }
}

File: Java/libraries/recognizers-text-choice/src/main/java/com/microsoft/recognizers/text/choice/config/BooleanParserConfiguration.java
package com.microsoft.recognizers.text.choice.config;

import com.google.common.collect.ImmutableMap;
import com.microsoft.recognizers.text.choice.Constants;

import java.util.Map;

public class BooleanParserConfiguration implements IChoiceParserConfiguration<Boolean> {

    public static Map<String, Boolean> Resolutions = ImmutableMap.<String, Boolean>builder()
            .put(Constants.SYS_BOOLEAN_TRUE, true)
            .put(Constants.SYS_BOOLEAN_FALSE, false)
            .build();

    @Override
    public Map<String, Boolean> getResolutions() {
        return Resolutions;
    }
}